docs/pages/repo/docs/reference/command-line-reference/login.mdx

```diff
@@ -7,20 +7,6 @@ description: Turborepo CLI Reference for `login` command
 
 Connect machine to your Remote Cache provider. The default provider is [Vercel](https://vercel.com/).
 
-### Options
-
-#### `--url`
-
-`type: string`
-
-Defaults to `https://vercel.com/`.
-
-#### `--api`
-
-`type: string`
-
-Defaults to `https://vercel.com/api`.
```

Comment:
`--api` should be a real option, but I just tried and it doesn't seem to do anything 🤔

Reply:
Agree that it's also something that I'm expecting to work, but I wasn't able to make to work either when I created this PR.